pycbc/io/hdf.py

```diff
@@ -835,3 +835,62 @@ def get_chisq_from_file_choice(hdfile, chisq_choice):
         raise ValueError(err_msg)
     return chisq
 
+def save_dict_to_hdf5(dic, filename):
```

Comment:
Where is this called from? Is that not implemented yet? Just checking you are missing commits.